### DISCUSSION TITLE
English sentence for 素知らぬ顔 sounds unnatural

### DISCUSSION THREAD

**Discussion by Morgawr** [Jitendex/Jitendex#50] [Errors and Corrections]
JP: 彼女はそ知らぬ顔をした。

EN: She put on an assumption of ignorance.

---

"An assumption of ignorance" sounds weird to me. Googling for that phrase verbatim returns results for only this example sentence and a bunch of other unrelated links (some law-related stuff and some scientific articles). 

I think something like "She put on an expression of feigned ignorance" sounds more natural and easier to understand.

**Comment by stephenmk** [comment]
Thanks. Probably best to keep it simple and use something like "She feigned ignorance." That's how the references below handle it.

---

Saitō
> 素知らぬ顔をする
> to feign ignorance―pretend not to Know―put on an innocent air

Progressive
> 擦れ違ったとき彼女は素知らぬ顔をして行ってしまった
> She 「cut me [ignored me ／pretended not to know me] when she passed me.
> 
> 先生が窓ガラスを割ったのは誰かときいたときに，秀夫は素知らぬ顔をしていた
> When the teacher asked who had broken the window, Hideo 「pretended ignorance [feigned innocence ].

Kenkyusha Shinwaei Daijiten
> 知らん顔をして
> unconcernedly; nonchalantly; with an air of perfect nonchalance; 〔何もなかったように〕 as if nothing had happened; 〔そ知らぬ顔で〕 as if one knew nothing about it
